java-chassis-dependencies/pom.xml

```diff
@@ -51,9 +51,9 @@
     <narayana.version>5.3.2.Final</narayana.version>
     <cxf.version>3.1.6</cxf.version>
     <logback.version>1.1.7</logback.version>
-    <brave.version>4.13.1</brave.version>
-    <zipkin.version>2.4.2</zipkin.version>
-    <zipkin-reporter.version>2.2.2</zipkin-reporter.version>
+    <brave.version>5.1.0</brave.version>
```

Comment:
need to change versions in java-chassis-distribution/src/release/LICENSE too.

Reply:
OK , thanks for your reminder. I will change versions in java-chassis-distribution/src/release/LICENSE.